brambling/templates/brambling/event/organizer/update.html

```diff
@@ -1,9 +1,295 @@
+{% load floppyforms %}
```

Comment:
Pretty sure line being above the `extends` tag is causing the following problem for me:

```
TemplateSyntaxError at /demo/glodnom/edit/

<ExtendsNode: extends 'brambling/event/organizer/__narrow.html'> must be the first tag in the template.
```